src/main/kotlin/app/RepoHasher.kt

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.model.Commit
+import app.model.Repo
+import app.utils.RepoHelper
+import io.reactivex.Observable
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import java.io.File
+import java.io.IOException
+
+/**
+ * RepoHasher hashes repository and uploads stats to server.
```

Comment:
Yeah, it's the language thing we agreed up. We say that 'Sourcerer hashes your repos into a profile' to stress the fact that we don't upload code anywhere.